client/src/language/languageClient.ts

```diff
@@ -154,10 +154,10 @@ export async function activateLanguageServer (context: ExtensionContext, bitBake
 }
 
 export async function deactivateLanguageServer (client: LanguageClient): Promise<void> {
-  if (client === undefined) {
-    return undefined
-  }
-  await client.stop()
+  await Promise.all([
+    embeddedLanguageDocsManager.deleteEmbeddedLanguageDocsFolder(),
```

Comment:
What happens if two separate VSCode sessions are running and one is closed? If I understand properly, this.embeddedLanguageDocsFolder is only for the session at hand so only it is deleted.

Reply:
I don't think it is possible to open the same workspace twice. VS Code has the option `--new-window` to `Force to open a new window.`. Even when using it (`code . --new-window`), I still can't open the same workspace twice.